documents/policies/internal/Working Group Formation.md

```diff
@@ -0,0 +1,42 @@
+# WCA Working Group Formation Policy
+
+### Version 1.0 {.version}
+
+## Purpose
+The purpose of this policy is to provide guidelines for the formation and operation of working groups within the World Cube Association (WCA) in order to facilitate collaboration, innovation, and progress in specific focus areas.
+
+## Policy
+1. Definition of a Working Group
+   1. A working group is a temporary team of WCA Staff and/or community members established to address a specific project or area of interest within the organization. Working groups are tasked with generating ideas, providing recommendations, and executing initiatives related to their assigned focus area
+2. Formation
+   1. Working groups may be proposed by any WCA Staff member.
+   2. The Working group must be sponsored by a team leader, a Senior Delegate or a Board member.
+   3.  Proposals for working groups should be submitted in writing to the Board of Directors for review and approval.
+3. Approval Process
+   1. The Board of Directors will review proposed working groups and assess their alignment with the organization's mission, goals, and available resources.
+   2. Approval will be granted based on the demonstrated need for the working group and its potential to positively impact the organization.
+4. Leadership
+   1. Each working group shall have a designated leader responsible for coordinating meetings, setting agendas, and ensuring progress towards established goals.
+   2. The leader will serve as the primary point of contact between the working group and the Board of Directors.
+5. Composition
+   1. Each working group shall consist of a minimum of four (4) members, including a designated leader. 
+   2. The leader should determine the most appropriate membership for the specific working group.  The leader should consider if there should be representation from particular WCA teams or committees or if membership should be open to all WCA staff.
+   3. Priority may be given to individuals with relevant expertise or experience. 
+6. Diversity and Inclusion
+   1. The WCA is committed to fostering a diverse and inclusive environment. When forming working groups, efforts should be made to ensure representation from individuals from diverse backgrounds, with a wide range perspectives, and experiences. This includes but is not limited to factors such as race, ethnicity, gender, age, sexual orientation, abilities, and socio-economic status. 
```

Comment:
Should say "a wide range *of* perspectives" here

Reply:
fixed